crates/matrix-sdk-search/src/index.rs

```diff
@@ -56,8 +56,8 @@ pub struct RoomIndex {
     schema: RoomMessageSchema,
     query_parser: QueryParser,
     room_id: OwnedRoomId,
-    uncommited_adds: HashSet<OwnedEventId>,
-    uncommited_removes: HashSet<OwnedEventId>,
+    uncommitted_adds: HashSet<OwnedEventId>,
+    uncommitted_removes: HashSet<OwnedEventId>,
```

Comment:
Would've loved to see small bits like these in separate commits, there's quite a few in here.